srcts/src/initialize/index.ts

```diff
@@ -2,15 +2,14 @@ import { disableFormSubmission } from "./disableForm";
 import { trackHistory } from "./history";
 import { determineBrowserInfo } from "./browser";
 
-import { windowShiny } from "../window/libraries";
-import { setShiny } from "../shiny";
+import { Shiny } from "../shiny";
 import { setUserAgent } from "../utils/userAgent";
 import { windowUserAgent } from "../window/userAgent";
 
 import { initReactlog } from "../shiny/reactlog";
 
 function init(): void {
-  setShiny(windowShiny());
+  window.Shiny = window.Shiny || new Shiny();
```

Comment:
I think this captures what the previous `setShiny()` function was doing, but I am not 100% sure.

Reply:
One reason that we previously didn't want to simply overwrite the `window.Shiny` object was because some other library might create a `window.Shiny = {}` object and then populate some of the properties.

This is the old code in this repo that created the `window.Shiny` object this way. https://github.com/rstudio/shiny/blob/v1.4.0/srcjs/_start.js 

```js
  var exports = window.Shiny = window.Shiny || {};
```


However, in practice it looks like, at least for shiny-server-client, its JS must get loaded after shiny.js, so the change in the PR shouldn't cause any behavioral changes:
https://github.com/rstudio/shiny-server-client/blob/92cbc96583f958f4e379120323fb3a48c2e61fd5/lib/main.js#L167-L187

Reply:
From a conversation with @schloerke: It would be better to just print a console error message if `window.Shiny` already exists at this point.